DigitalLearningSolutions.Web/Controllers/MyAccountController.cs

```diff
@@ -415,5 +468,19 @@ DlsSubApplication dlsSubApplication
 
             return delegateAccount is { Active: true } ? delegateAccount : null;
         }
+
+        private async Task LogOutAndLogIntoCentrelessAccount(UserAccount userAccount)
+        {
+            var claims = LoginClaimsHelper.GetClaimsForCentrelessSignIn(userAccount);
+            var claimsIdentity = new ClaimsIdentity(claims, "Identity.Application");
+            var authProperties = new AuthenticationProperties

```

Comment:
You should check whether remember-me has been ticked too. I think the property is called `IsPersistent` or something. Preserve that in the AuthenticationProperties.